presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java

```diff
@@ -283,6 +293,17 @@ public FragmentProperties setHashPartitionedOutput(List<Symbol> partitionKeys, O
             return this;
         }
 
+        public FragmentProperties setNullEverywhere()
+        {
+            if (!Optional.of(OutputPartitioning.HASH).equals(outputPartitioning)) {
```

Comment:
Use `checkState(outputPartitioning.isPresent() && outputPartitioning.get().equals(OutputPartitioning.HASH), ...` as that's the more common pattern in our codebase